### PR TITLE
Merge release

### PR DESCRIPTION
**Pull request by panjan**

### CODE REVIEW

File: lib/gooddata/lcm/actions/synchronize_user_filters.rb [hunk 1]
```diff
@@ -232,10 +233,10 @@ def call(params)
               working_client_ids << client_id
               filters_to_load = GoodData::UserFilterBuilder.get_filters(new_filters, symbolized_config)
               puts "Synchronizing #{filters_to_load.count} filters in project #{project.pid} of client #{client_id}"
-              project.add_data_permissions(filters_to_load, run_params)
+              partial_results = project.add_data_permissions(filters_to_load, run_params)
```
Comment (yenkins) [inline lib/gooddata/lcm/actions/synchronize_user_filters.rb]: Calls 'project.add_data_permissions(filters_to_load, run_params)' 6 times (DuplicateMethodCall)

File: lib/gooddata/models/user_filters/user_filter_builder.rb [hunk 2]
```diff
@@ -387,6 +387,7 @@ def self.execute_mufs(user_filters, options = {})
       end
 
       if dry_run
+        GoodData.logger.warn('Option "dry_run" specified. No user filters will be altered!')
```
Comment (yenkins) [inline lib/gooddata/models/user_filters/user_filter_builder.rb]: Calls 'gooddata.logger' 3 times (DuplicateMethodCall)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/212/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1054/ : SUCCESS in 1m 11s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1323/ : SUCCESS in 1m 16s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/213/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1055/ : SUCCESS in 1m 12s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1324/ : SUCCESS in 1m 14s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/74/ : SUCCESS in 16s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1056/ : SUCCESS in 1m 26s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1325/ : SUCCESS in 1m 12s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1427/ : ABORTED in 12m 30s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-lcm-integration-tests-zuul-el7/12/ : FAILURE in 28s

**Comment by panjan** [comment]
tests passed: https://travis-ci.org/panjan/gooddata-ruby/builds/388768382

**Pull request merged**